packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -26,18 +29,24 @@ export const useFeedLayout = ({
   feedName: feedNameProp,
   feedRelated = true,
 }: UseFeedLayoutProps = {}): UseFeedLayout => {
+  const hasCommentFeed = useFeature(feature.commentFeed);
```

Comment:
I think this `useFeature` should be replace with `withExperiment` on `CommentFeed` and then only allocate if users is on `discussions` page.

Reply:
The hook should only return true if user is on discussions. This will prevent user being allocated on any other place where `useFeedLayout` is used.

Reply:
Hmm not sure, as it's also a sidebar condition?
Allocation would happen on mainLayout pretty much guaranteed anyway.

Reply:
Yeah, since it is sidebar it is less relevant but I think cleaner vs adding dependencies to hook that can technically be used anywhere.

Reply:
Yeah agreed.
Think technically probably best to just have separate hook even.
Let's see how this experiment evolves

Reply:
Solved it on slack. There were some other dependencies.